packages/@coorpacademy-components/src/molecule/learning-priority-modal/test/fixtures/default.js

```diff
@@ -0,0 +1,66 @@
+export default {
+  props: {
```

Comment:
Could you had some different fixtures to showcase the different states?

like filter selected; no certificates; already selected MP; filter used; filter not matching